website/source/docs/builders/qemu.html.md.erb

```diff
@@ -112,8 +112,8 @@ Linux server and have not enabled X11 forwarding (`ssh -X`).
 ### Optional:
 
 -   `accelerator` (string) - The accelerator type to use when running the VM.
-    This may be `none`, `kvm`, `tcg`, `hax`, or `xen`. The appropriate software
-    must have already been installed on your build machine to use the
+    This may be `none`, `kvm`, `tcg`, `hax`, `hvf`, or `xen`. The appropriate
+    software must have already been installed on your build machine to use the
```

Comment:
Can you add a note that hvf is experimental? I'm always wary of just adding support for experimental stuff without hedging that there may be issues in the underlying tool.